packages/amplify-e2e-core/src/init/deleteProject.ts

```diff
@@ -2,6 +2,9 @@ import { nspawn as spawn, retry, getCLIPath, describeCloudFormationStack, getPro
 
 export const deleteProject = async (cwd: string, profileConfig?: any): Promise<void> => {
   const { StackName: stackName, Region: region } = getProjectMeta(cwd).providers.awscloudformation;
+  if (!stackName || !region) {
```

Comment:
Was it a missing check causing test failures or is it due to changes in init?

Reply:
We have some tests that just do an init and then check some local state but then the afterEach step tries to delete the project and fails here because the stack doesn't exist yet